### DISCUSSION TITLE
bit of a harsh outcome

### DISCUSSION THREAD

**Discussion by Shibva** [HbmMods/Hbm-s-Nuclear-Tech-GIT#647] [General]
The following is in regards to the Fandom Wiki page; Due to the circumstances that remain unknown with discord, this was the only other place where I can get into contact. 

Ok, I admit that some of my edits may have been counterproductive but an infinite edit ban nostalgia? ya could have just told me to avoid doing that sort of thing and I would have gotten the message; after all, I do love this mod and want to contribute in any way I can. Its also a bit harsh that I was given an infinite duration for my first offense, It was not like I was deleting entire pages. 

so yeah, can this be looked into; I mean tbf I dint really see any wiki editing guidelines for certain do's and dont's. AT least now I understand some of the don't at least

**Comment by Vaern** [comment]
There can be more reasons factored into a ban, rather than it just being the first offense.

**Reply by Shibva** [reply]
it was on the grounds of including what was seen as unessisary blabber; though in retrospect I can see it now, though the fact that its duration was set to infinite was a bit harsh, I mean I honestly thought I was contributing. Guess I was not aware that some pages are off-limits for commoners to edit

this is also getting me to a because a minor edit on the FEL page I made needs to be corrected. for some reason, I entered a M into the second buliten and its bugging me that I can fix my typo

**Reply by Shibva** [reply]
even though the community seems to have mixed opinions on me (those that even remember me) I love this mod and would never do anything to impeed or hurt it in any way. I may be bias but its one of the most unique all-around technical mods I ever played

**Comment by Adam29Adam29** [comment]
Sir, you been known to abuse the open issue system by suggesting (even "improovements" are suggestions) unnecessary or completely braindead things and they will be ignored. And along, about your community opinion, you are known as the bad suggestion GitHub user. Take it as you want but if you had some standards (which I see there is lack of), you wouldnt be in this scenario begging why you were banned of the wiki when you got history of doing more negative things.

**Reply by Shibva** [reply]
you think I don't know that already? ya know I've stopped doing that after someone gave me some wise words. I will admit some of them are in fact braindead and looking back on those actions I couldn't agree more. I will admit that my standards are all over the damn place, but maybe its time to try to get them straight.

im not who I was all those years ago, I've grown and matured. Regardless of the past I still want to help in any way I can possibly offer, even if its nothing more than working on wiki entries. I mean its better than trying to be a good-idea generator and is a step in the right direction

**Reply by ncr-t** [reply]
[Im not who I was all those years ago] with your activity in the recent months, we have the total right to doubt that, and proof that you haven't.

**Reply by Adam29Adam29** [reply]
![IMG_20220624_223825](https://user-images.githubusercontent.com/92393622/176029917-42480f33-5216-45e7-86af-173c59419e89.jpg)

**Reply by Adam29Adam29** [reply]
Can't you stop roll crying the floor, can you?

**Reply by Shibva** [reply]
> [Im not who I was all those years ago] with your activity in the recent months, we have the total right to doubt that, and proof that you haven't.

I was referring to other things when I said that.

**Comment by Nos74** [comment]
k        e         k

**Reply by Shibva** [reply]
so yeah, im hoping I can continue to help out in this small way I can. though if I am allowed to I do ask if there's any sort of guidlenines or template to follow when making new entiries

I ask because I was going to make one regarding the new weapon. I know now at the least to avoid random nonsense. 

also sorry about that random stuff, now I know to avoid that sort of thing

**Comment by Shibva** [comment]
Oh yeah, because it dosent matter Id figure I explain that one edit in secrets

the reason why I made a Llamas with hats reference was actually because of a unique circumstance. you see, I learned that null, or the digamma rocket as its actually called could only be fired by one of the lauchers, a laucher that I happened to make a joke on to one of my friends

that laucher is none other that Kaurl 

![image](https://user-images.githubusercontent.com/38770780/176049447-5f25dde0-195b-40c5-b03f-fe63cce329ed.png)

**Reply by Shibva** [reply]
wrong place, ill admit that in retrospect. but being cryptic, im a bit proud of that, and yes, I cant help but smirk even though I know damn well that I shoudlnt
